javadoc-parser/src/main/java/com/github/chhorz/javadoc/tags/DeprecatedKotlinTag.java

```diff
@@ -0,0 +1,39 @@
+/**
+ *
+ *    Copyright 2018-2022 the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.github.chhorz.javadoc.tags;
+
+/**
+ * <a href="https://kotlinlang.org/docs/kotlin-doc.html#kdoc-syntax">Kotlin documentation</a>
+ *
+ * @author chhorz
+ */
+public class DeprecatedKotlinTag extends StructuredTag {
```

Comment:
This tag does not exist. Need to remove this class

Reply:
This class was used for the KDoc `@Deprecated` tag. I will try if i could merge the Javadoc and KDoc classes.

Reply:
@chhorz You misunderstood. Read again, I specifically added a screenshot with the documentation. let's do it again:
```
KDoc does not support the @deprecated tag. Instead, please use the @Deprecated annotation.
```
It says here: kotlin does **NOT support** the `@deprecated` tag.

It is proposed to use the **ANNOTATION** `@Deprecated` instead. Annotation is not a tag

Reply:
Indeed i misred the documentation. I will remove the tag for KDoc.